frontend/api/admin.ts

```diff
@@ -34,16 +34,30 @@ export class Admin {
     }
   }
 
+  /**
+   * get an {@link Admin} data.
+   *
+   * @param {string} token
+   * @param {string} email
+   * @returns {Promise<string[]>}
+   */
+  static async get(email: string, token: string): Promise<string[]> {
+    let json = Api.getAuth(`/admin/${email}`, token);
+    return json;
+  }
+
   /**
    * login an {@link Admin}.
    *
    * @param {string} email - email of Admin
    * @param {string} password - password of Admin
-   * @returns {Promise<string>} JSON Web Token authenticating the admin
+   * @returns {Promise<object>} JSON Web Token authenticating the admin
    */
-  static async login(email: string, password: string): Promise<string> {
-    let json = Api.post("/login", { email: email, password: password });
-    return json["token"];
+  static async login(email: string, password: string): Promise<object> {
```

Comment:
Login right now returns just a token, are you looking for it to emulate your `register` return value? I think it's best to be explicit as possible in the typing, for instance object will be

```typescript
{token: string, user: { name: string, email: string, superUser: boolean, _id: string }}
```

I still think we should scrap `_id` on the backend, it's just not useful for us.
```